tests/unit/src/kakarot/accounts/eoa/test_library.py

```diff
@@ -1,5 +1,6 @@
 import pytest
 import pytest_asyncio
+from collections import defaultdict
```

Comment:
sorry for this, I meant `Counter` see for example

```
from collections import Counter
d = Counter()
d["0x1234"] += 2134
```

Reply:
Ah nice, I didn't know about this will use this next time :)